app/src/main/java/com/github/freeman/bootcamp/MainMenuActivity.kt

```diff
@@ -134,6 +136,17 @@ fun PlayButton() {
     )
 }
 
+@Composable
+fun DisplayRecapsButton() {
+    val context = LocalContext.current
+    MainMenuButton(
+        testTag = "displayRecapsButton",
```

Comment:
Maybe this testTag string should be extracted and put in` strings.xml`, but it hasn't been done before for the other buttons, so I guess it's consistent with the conventions in this file.